integration/kubernetes/e2e_conformance/run.sh

```diff
@@ -109,7 +109,7 @@ run_sonobuoy() {
 		fi
 	fi
 	echo "running: ${cmd}"
-	eval "${cmd}"
+	eval "${cmd}" || die "Sonobuoy run failed"
```

Comment:
Here I am not sure the script should exit or if it should try to retrieve the results. It might have failed due to timeout, in case there are unfinished tests yet.

Reply:
yeah - I think the same, you should let it run :smile:

Reply:
Ah, I prefer returning earlier in case of failure.

Reply:
I think that in case there are some failed tests, this command will return non-zero and in that case we won't be able to retrieve the results for analysis, so I would not make it `die` here.

Reply:
@chavafg good point. I confess I could not run sonobuoy so I wasn't sure a test failure would end up on the process returning non-zero code. So I will remove this statement, rebase the code and hopefully be able to merge it :)